src/app/search/directives/search-series-form.component.css

```diff
@@ -1,3 +1,7 @@
+form {
+  display: block;
```

Comment:
No, forms.css makes them display:flex. I haven't verified that's actually being used though. This file needs to go away eventually though as it's very generic styling that belongs in the styleguide.